api-specs/api_spec_PGS.yaml

```diff
@@ -362,6 +362,97 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /request-payments/xpay:
+    post:
+      summary: payment authorization request to xPay
+      tags:
+        - payment-transactions-controller
+      operationId: auth-request-xpay
+      parameters:
+        - in: header
+          name: X-Client-ID
+          description: channel origin (APP/Web)
+          example: APP
+          schema:
+            type: string
+          required: true
+        - in: header
+          name: MDC-Fields
+          description: MDC information
+          example: 97g10t83x7bb0437bbc50sdf58e970gt
+          schema:
+            type: string
+          required: false
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/XPayAuthRequest'
+        required: true
+      responses:
+        "200":
+          description: OK
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/XPayAuthResponseEntity'
+        "400":
+          description: Bad request - missing mandatory parameters
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/XPayAuthResponseEntity400'
+        "401":
+          description: transactionId already processed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/XPayAuthResponseEntity401'
+        "500":
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/XPayAuthResponseEntity500'
+  /request-payments/xpay/auth/{requestId}:
```

Comment:
Because in this case we have 2 GET calls, the first GET (`/request-payments/xpay/auth/{requestId`}) is used to check the XPay authorization, the second GET (`/request-payments/xpay/auth/{requestId}`) is used to verify the XPay payment.

Reply:
I'm not sure, but we'll review it next refinement @gammam